### ISSUE TITLE
fix "there is no user with the name ..."

### ISSUE THREAD

**Issue by maro-21** [openstreetmap/openstreetmap-website#3892]
https://www.openstreetmap.org/user/kkkkkkklllll
_Sorry, there is no user with the name %{user}. Please check your spelling, or maybe the link you clicked is wrong._

In many cases, when someone sees this message, the link was good, only the user in the meantime deleted the account.

So I suggest to changed it to:
_Sorry, there is no user with the name %{user}. Maybe a wrong link was used or the user deleted the account._
or similar.

**Comment by tomhughes** [comment]
Well if the user has deleted (or renamed) their account then the link by definition wasn't good.

**Comment by maro-21** [comment]
At the time someone provided the link, it was a good one. It's just that this person didn't click on it in time and the link became overdue.

**Comment by matkoniecz** [comment]
> Well if the user has deleted (or renamed) their account then the link by definition wasn't good.

Yes, but message is confusing to people who tried to follow link. For example there was some problematic spammer, user was deleted and later someone was asking to provide actual link to user. They thought that initial complainer mangled link somehow, unaware that account was removed already.
